### ISSUE TITLE
Turn off hdmi to conserver power

### ISSUE THREAD

**Issue by tkurki** [tkurki/marinepi-provisioning#21]
https://glframebuffer.wordpress.com/2013/08/28/raspberrypi-how-to-turn-off-hdmi-from-raspberry-pi/

**Comment by hkapanen** [comment]
Is this only about conserving the power of an attached monitor, or do you know if it also reduces the power consumption of the Pi's video core? If the latter is true, then this would make sense to headless setups as well.

**Comment by joux3** [comment]
[This discussion](http://raspberrypi.stackexchange.com/questions/43285/raspberry-pi-3-vs-pi-2-power-consumption-and-heat-dissipation/47840#47840) would hint at less power being consumed for headless setups. Anybody care to measure it?

**Comment by tkurki** [comment]
Yep, power saving in headless setup was the point here. The issue description was pretty minimal, sorryyyy.

**Comment by hkapanen** [comment]
OK - I can see a benefit in order of 10mA (0.28->0.27A) on a headless RPI3 idling on a WLAN. No more digits available from my el-cheapo USB power meter.
